Slimefun Beautified/pack.mcmeta

```diff
@@ -1,6 +1,6 @@
 {
    "pack":{
       "pack_format":9,
-      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§e(24/10/2022 §7- &ev1.1)"
+      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§e6a588bd §7- &ev1.0)"
```

Comment:
Git commit ID is not from this repository, will be changed to match version bump in the recent future from this repository. Odd `&e` placed before `v1.0` . I assume an attempt at adding color? Personally I don't believe it goes well adding color to the version number.

Preview of oddity:
![image](https://user-images.githubusercontent.com/26176694/197628090-c8282c2e-ea29-48f4-b032-14a6a9385a5a.png)

```suggestion
      "description":"§a§lSlimefun §5§lB§d§le§9§la§1§lu§3§lt§b§li§a§lf§e§li§6§le§c§ld \n§eUnknown §7- &v1.0)"
```